VideoLocker/res/layout/header_discussion_topics_list.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:background="@color/white"
+    android:orientation="vertical">
+
+    <org.edx.mobile.view.custom.ETextView
+        android:id="@+id/discussion_topic_all_posts"
+        style="@style/regular_grey_text"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:background="?android:attr/selectableItemBackground"
+        android:ellipsize="end"
+        android:gravity="center_vertical"
+        android:padding="@dimen/edx_margin"
+        android:singleLine="true"
+        android:text="@string/discussion_posts_filter_all_posts"
+        android:textDirection="locale"
```

Comment:
Do we need to set the `textDirection` to `locale` for all `TextView` items? If we're using an RTL language, then it should automatically align the text to the right in the default `firstStrong` heuristic. If we actually want to change the `textDirection` globally to `locale`, then it should probably be done in the base style for `TextView`.